terraform/aws/README.md

```diff
@@ -0,0 +1,141 @@
+
+## Create MOSIP Infrastructure 
+
+### Prerequisites
+* Create a new branch `env-<environment_name>` from master/develop branch.
+* Goto `terraform/aws` location and update environment related details in `env.tfvars` file.
+
+### Run `terraform plan / apply` workflow to set up MOSIP infrastructure
+* This GitHub Action automates the Terraform workflow,
+  allowing users to run `terraform plan` and optionally `terraform apply` commands within a CI/CD pipeline.
+  The workflow is triggered manually via workflow_dispatch.
+* To trigger this workflow, go to the `Actions` tab in your GitHub repository. Select `terraform plan / apply` workflow and provide the required inputs and click `Run workflow` to start the workflow.
+  ![terraform-1.png](../../docs/_images/terraform-1.png)
+
+### Inputs
+* `SSH_PRIVATE_KEY (required)`: GitHub secret name containing the private key for SSH login on the nginx node.
+* `TERRAFORM_APPLY (optional)`: Boolean flag to apply the Terraform plan. Defaults to false.
+
+### Environment Variables
+* `AWS_ACCESS_KEY_ID`: The AWS access key ID used for authenticating with AWS services. This is stored as a GitHub secret `AWS_ACCESS_KEY_ID`.
+* `AWS_SECRET_ACCESS_KEY`: The AWS secret access key corresponding to the AWS access key ID. This is stored as a GitHub secret `AWS_SECRET_ACCESS_KEY`.
+* `TF_VAR_SSH_PRIVATE_KEY`: The SSH private key used for accessing nginx server. This is referenced from the input `SSH_PRIVATE_KEY` and stored as a GitHub secret.
+
+## Destroy MOSIP Infrastructure
+
+### Run `terraform destroy` workflow 
+* This GitHub Action automates the `Terraform destroy` command within a CI/CD pipeline.
+The workflow can be manually triggered to destroy infrastructure managed by Terraform.
+* To trigger this workflow, go to the `Actions` tab in your GitHub repository. Select `terraform destroy` workflow and provide the required inputs and click `Run workflow` to start the workflow.
+  ![terraform-destroy-1.png](../../docs/_images/terraform-destroy-1.png)
+
+### Inputs
+* `SSH_PRIVATE_KEY (required)`: GitHub secret name containing the private key for SSH login on the nginx node.
+* `TERRAFORM_DESTROY (optional)`: Boolean flag to determine whether to execute the Terraform destroy command. Defaults to false.
+
+### Environment Variables
+* `AWS_ACCESS_KEY_ID`: The AWS access key ID used for authenticating with AWS services. This is stored as a GitHub secret `AWS_ACCESS_KEY_ID`.
+* `AWS_SECRET_ACCESS_KEY`: The AWS secret access key corresponding to the AWS access key ID. This is stored as a GitHub secret `AWS_SECRET_ACCESS_KEY`.
+* `TF_VAR_SSH_PRIVATE_KEY`: The SSH private key used for accessing nginx server. This is referenced from the input `SSH_PRIVATE_KEY` and stored as a GitHub secret.
+
+# Terraform Setup for MOSIP Infrastructure
+
+## Overview
+This Terraform configuration script set up the infrastructure for MOSIP (Modular Open Source Identity Platform) on AWS.
```

Comment:
MOSIP (Modular Open Source Identity Platform) on AWS --> MOSIP deployment on AWS.

Reply:
I dont see this is taken care. please correct.